docker/entrypoint.sh

```diff
@@ -5,10 +5,6 @@ if [ "$DB_AUTO_MIGRATE" == "true" ]; then
   bin/rake db:migrate
 fi
 
-if [ "$TEST_DB_AUTO_MIGRATE" == "true" ]; then
```

Comment:
we're just loading schema now for tests, so shouldn't need to worry about test db migrations